clients/src/test/java/org/apache/kafka/clients/producer/internals/TransactionManagerTest.java

```diff
@@ -112,12 +112,21 @@
     private static final int MAX_BLOCK_TIMEOUT = 1000;
     private static final int REQUEST_TIMEOUT = 1000;
     private static final long DEFAULT_RETRY_BACKOFF_MS = 100L;
+
+
     private final String transactionalId = "foobar";
     private final int transactionTimeoutMs = 1121;
 
     private final String topic = "test";
     private final TopicPartition tp0 = new TopicPartition(topic, 0);
     private final TopicPartition tp1 = new TopicPartition(topic, 1);
+    private final long producerId = 13131L;
```

Comment:
Extracting all the common parameters for easier test composition, and easier templating for the downgrade test, so that we don't need to carry these redundant messages around.